### ISSUE TITLE
500 error when creating file with wopi 10.3.0

### ISSUE THREAD

**Issue by ScharfViktor** [owncloud/ocis#8272]
maybe it related https://github.com/owncloud/ocis/pull/8207

after wopi updating `WOPISERVER_DOCKER_TAG:-v10.3.0` I cannot create file

Steps: 
- using https://github.com/owncloud/ocis/blob/master/deployments/examples/ocis_wopi/docker-compose.yml up ocis with wopi
- try to create new `.odt` or .`docx` file

Actual: 500 error

ocis_logs: 
```
2024-01-23 15:06:29 {"level":"info","service":"proxy","proto":"HTTP/1.1","request-id":"bd79b16b-cfc4-412d-87c1-29a41a50f50c","traceid":"8b339d728fc52a942687f40926e1b2e7","remote-addr":"192.168.65.1","method":"POST","status":500,"path":"/app/new","duration":47.218917,"bytes":70,"time":"2024-01-23T14:06:29Z","line":"github.com/owncloud/ocis/v2/services/proxy/pkg/middleware/accesslog.go:34","message":"access-log"}

2024-01-23 15:06:29 {"level":"error","service":"storage-users","pkg":"rhttp","traceid":"898db91cb1a21447dd87456b27a1a24e","host":"127.0.0.1","method":"PUT","uri":"/data/simple/beb47e83-2863-4d34-b5b4-5bd8909bcd31","url":"/beb47e83-2863-4d34-b5b4-5bd8909bcd31","proto":"HTTP/1.1","status":500,"size":0,"start":"23/Jan/2024:14:06:29 +0000","end":"23/Jan/2024:14:06:29 +0000","time_ns":281292,"time":"2024-01-23T14:06:29Z","line":"github.com/cs3org/reva/v2@v2.18.1-0.20240122171534-e8fc07f2395a/internal/http/interceptors/log/log.go:112","message":"http"
```

but after reload page -> file is create and can be open

https://github.com/owncloud/ocis/assets/84779829/dd5ae392-169f-4d26-b42b-e5f0d255a0c2

cc @aduffeck

**Comment by micbar** [comment]
@ScharfViktor Good qualification. Adding label.

**Comment by micbar** [comment]
Fixed by #8287 due to reva update.
